ml-core-base/api/ml-core-base.api

```diff
@@ -0,0 +1,7 @@
+public final class com/stripe/android/mlcore/base/BuildConfig {
```

Comment:
Probably makes the most sense to turn the generation of this class off.

In build.gradle 

```
android {
  buildFeatures {
    buildConfig = true
  }
}
```